docker-build-package

```diff
@@ -106,7 +114,7 @@ if [ "$arch_indep" = "true" -a "$ARCH" != "amd64" ]; then
     exit 0
 fi
 
-echo "Building $recipe_name..."
+echo "Building $recipe_name... $*"
```

Comment:
Thank you. Good idea.

I suggest thought a separate commit for it and something more explicit like
```suggestion
echo "Building $recipe_name with arguments: $*"
```

Reply:
fixed